recipes/sqlitecpp/all/conanfile.py

```diff
@@ -19,11 +19,13 @@ class SQLiteCppConan(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "lint": [True, False, "deprecated"],
+        "omit_load_extension": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
         "lint": "deprecated",
+        "omit_load_extension": False,
```

Comment:
this does not exist in version 2? https://www.sqlite.org/src/file?name=src/os.c&ci=50d7e50a96907eeb
It should be deleted in `configure`

[ref v3](https://github.com/sqlite/sqlite/blob/cf145047b0d0ef57ffaaaad3e7d17dcf653a3244/src/os.c#L253)

Reply:
yes, it exists in v2:
https://github.com/SRombauts/SQLiteCpp/blob/sqlitecpp-2.x/src/Database.cpp#L214

Reply:
https://github.com/SRombauts/SQLiteCpp/blob/ee37e4eb007fddf43f54b72740c467a1469d3425/CMakeLists.txt#L212

Reply:
@ericLemanissier comment pointed out my mistake... I was looking at sqlite not the c++ wrapper

Reply:
well, when I checked this PR I thought was about sqlite (C lib) :sweat_smile: